spring-integration-jdbc/src/main/resources/org/springframework/integration/jdbc/schema-sybase.sql

```diff
@@ -27,3 +27,11 @@ CREATE TABLE INT_MESSAGE_GROUP  (
 	UPDATED_DATE DATETIME DEFAULT NULL,
 	constraint MESSAGE_GROUP_PK primary key (GROUP_KEY, REGION)
 ) LOCK DATAROWS;
+
+CREATE TABLE INT_LOCK  (
```

Comment:
I think we can leave existing scripts as is and create new in the separate package with a new appropriate name, e.g. `lock-schema-derby.sql` etc.

Although I can find your argument as fine, too. We provide the full script with all supported objects from our side. And that is target DBA task to choose the proper prefix and which objects to create or not. Or even do his own modifications.

So, let's leave as is and even combine all scripts (`store.channel` package) into one file in the `5.0` :smile: : https://jira.spring.io/browse/INT-4050

Not your task already. Although contribution welcome :smile: